go/internal/multipeer-connectivity-transport/driver/mc-driver/mc-driver.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why did you add a XCode workspace in the Go package?

Reply:
I comment only this file but this comment is for the entire folder

Reply:
> Why did you add a XCode workspace in the Go package?

Because that was that we did with the ble android package. I thought I should keep it because it's more easy to modify this package with the xcode workspace for everyone.

Reply:
In v1, the gradle files were included in the Android driver since they were necessary to compile it.

On the other hand, there was no XCode file in the BLE Android driver since XCode was not used and cgo compiles it directly. It is the same for the MPConnectivity driver.